README-de.md

```diff
@@ -37,6 +37,12 @@ Um die Kubernetes-Website lokal laufen zu lassen, empfiehlt es sich, ein speziel
 
 > Wenn Sie die Website lieber lokal ohne Docker ausführen möchten, finden Sie weitere Informationen unter [Website lokal mit Hugo ausführen](#Die-Site-lokal-mit-Hugo-ausführen).
 
+Das benötigte Docsy Hugo Theme muss als git submodule installiert werden:
+
+```
```

Comment:
```
#Installiere die Abhängigkeiten
yarn
```

Reply:
At least yarn is in the EN also mentioned

Reply:
Hi @hikhvar ,
The `en` version of the file has a pending update.
For further information see PR #22995 .